src/vmware/azext_vmware/tests/latest/recordings/test_vmware_datastores.yaml

```diff
@@ -0,0 +1,379 @@
+    headers:
+      cache-control:
+      - no-cache
+      content-length:
+      - '2031'
+      content-type:
+      - application/json; charset=utf-8
+      date:
+      - Mon, 15 Feb 2021 13:09:27 GMT
+      expires:
+      - '-1'
+      pragma:
+      - no-cache
+      server:
+      - nginx
+      strict-transport-security:
+      - max-age=31536000; includeSubDomains
+      transfer-encoding:
+      - chunked
+      vary:
+      - Accept-Encoding
+      x-content-type-options:
+      - nosniff
+    status:
+      code: 200
+      message: OK
+- request:
+    body: '{"properties": {"netAppVolume": {"nfsProviderIp": "10.50.1.4", "nfsFilePath":
+      "ANFVol1FilePath"}}}'
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - vmware datastore create
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '98'
+      Content-Type:
+      - application/json
+      ParameterSetName:
+      - --name --resource-group --cluster --private-cloud --nfs-file-path --nfs-provider-ip
+      User-Agent:
+      - AZURECLI/2.19.1 azsdk-python-avsclient/unknown Python/3.8.7 (Windows-10-10.0.18362-SP0)
+    method: PUT
+    uri: https://eastus2euap.management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rasivagu-cloudsan-rg/providers/Microsoft.AVS/privateClouds/rasivagu-sddc/clusters/Cluster-1/datastores/ANFDatastore1?api-version=2021-01-01-preview
+  response:
+    body:
+      string: '{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rasivagu-cloudsan-rg/providers/Microsoft.AVS/privateClouds/rasivagu-sddc/clusters/Cluster-1/datastores/ANFDatastore1","name":"ANFDatastore1","properties":{"netAppVolume":{"nfsFilePath":"ANFVol1FilePath","nfsProviderIp":"10.50.1.4"},"provisioningState":"Pending"},"type":"Microsoft.AVS/privateClouds/clusters/datastores"}'
```

Comment:
`Pending` is not in `DatastoreProvisioningState`.

Reply:
Sure Cameron, I'll investigate this.